src/libs/Network.js

```diff
@@ -31,15 +35,21 @@ let didLoadPersistedRequests;
 Onyx.connect({
     key: ONYXKEYS.NETWORK_REQUEST_QUEUE,
     callback: (persistedRequests) => {
-        if (didLoadPersistedRequests || !persistedRequests) {
-            return;
-        }
+        ActiveClientManager.isReady.then(() => {
+            // Only process queue when client is Leader and we have not retried the requests already
```

Comment:
Why should we only process the queue when the client is the leader? (Put it in the comment)